tests/dsl/dtd/dtd_test_ce.c

```diff
@@ -0,0 +1,683 @@
+#include <mpi.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <assert.h>
+#include <string.h>
+#include "parsec/parsec_comm_engine.h"
+
+#include "parsec/runtime.h"
+
+#define ACTIVE_MESSAGE_FROM_0_TAG 2
+#define ACTIVE_MESSAGE_FROM_1_TAG 3
+#define NOTIFY_ABOUT_GET_FROM_0_TAG 4
+#define NOTIFY_ABOUT_PUT_FROM_0_TAG 5
+#define NOTIFY_ABOUT_MEM_HANDLE_FROM_1_TAG 6
+
+int
+get_end(parsec_comm_engine_t *ce,
+        parsec_ce_mem_reg_handle_t lreg,
+        ptrdiff_t ldispl,
+        parsec_ce_mem_reg_handle_t rreg,
+        ptrdiff_t rdispl,
+        size_t size,
+        int remote,
+        void *cb_data);
+
+int
+put_end(parsec_comm_engine_t *ce,
+        parsec_ce_mem_reg_handle_t lreg,
+        ptrdiff_t ldispl,
+        parsec_ce_mem_reg_handle_t rreg,
+        ptrdiff_t rdispl,
+        size_t size,
+        int remote,
+        void *cb_data);
+
+int
+put_end_ack(parsec_comm_engine_t *ce,
+            parsec_ce_tag_t tag,
+            void *msg,
+            size_t msg_size,
+            int src,
+            void *cb_data);
+
+int counter = 0;
+int my_rank;
+
+/* Tag 0 for float message */
+int
+callback_tag_2(parsec_comm_engine_t *ce,
+               parsec_ce_tag_t tag,
+               void *msg,
+               size_t msg_size,
+               int src,
+               void *cb_data)
+{
+    (void) ce; (void) cb_data;
+    printf("[%d] In callback for tag %llu, message sent from %d size: %ld message: ", my_rank, tag, src, msg_size);
```

Comment:
Replace `%llu` with `%"PRIu64"

Reply:
Argh, those were debug statement that shouldn't have made it into the commit. We'll remove them